addon/initializers/setup-session.js

```diff
@@ -11,5 +11,7 @@ export default function setupSession(registry) {
     store = 'session-store:test';
     registry.register(store, Ephemeral);
   }
+
+  inject(registry, store, 'cookies', 'service:cookies');
```

Comment:
http://stackoverflow.com/questions/34970881/inject-a-service-into-an-ember-object-not-an-ember-controller

Specifically because the `session-stores` are Ember Objects. Ember Objects do not have lifecycle hooks, so `Ember.inject.service()` does not work.

We could refactor the `session-stores` to be services, but that expands the scope of this PR by a lot.